### ISSUE TITLE
A/libc: Fatal signal 11 (SIGSEGV), code 1 (SEGV_MAPERR),

### ISSUE THREAD

**Issue by Vivaan7** [realm/realm-js#5179]
### How frequently does the bug occur?

All the time

### Description

i am writing data into real m  after adding data my application is going to close and i got the crash stated what below line 

"A/libc: Fatal signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x6e754600000004 in tid 15221 (mqt_js), pid 15154"

 

### Stacktrace & log output

```shell
"A/libc: Fatal signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x6e754600000004 in tid 15221 (mqt_js), pid 15154"
```

### Can you reproduce the bug?

Not yet

### Version

10.6.1

### What SDK flavour are you using?

Local Database only

### Are you using encryption?

No, not using encryption

### Platform OS and version(s)

Android 12 (31 API)

### Build environment

none using Logs.

**Comment by Vivaan7** [comment]
@kneth  @takameyer   hello now i am getting crash after adding data this is unbelievable that we are getting this much issues. please do need full help.

**Comment by kneth** [comment]
@Vivaan7 Could you provide more information, including your dependencies and the full stack trace? Do you use Expo?

**Comment by Vivaan7** [comment]
@kneth  no i am using react native cli


  "realm": "^10.6.1",
  "react": "17.0.2",
  "react-native": "0.65.0",
  "react-native-reanimated": "^2.9.1",
  
  
  DBhelper.js
  import React from "react";
import Realm from "realm";
import { WOListingSchema } from "./Schema/WOListingSchema";
export default new Realm({
  path: "Autofacets.realm",
  schemaVersion: 1,
  schema: [WOListingSchema], //we can have multiple schemas here
});






//data insert file function 
MainFile.js  



//jsonData is array 
 writeDataToDB = (jsonData) => {
    let dataWritten = false;
    let errorMsg = undefined;

    try {
      Realm.write(() => {
        jsonData.forEach((object) => {
          let inputData = {
            [woListSchemaKeys.totalTimeSpent]: object.totalTimeSpent,
            [woListSchemaKeys.totalPlannedMechanicTime]:
              object.totalPlannedMechanicTime,
            [woListSchemaKeys.mechanicName]: object.mechanicName,
            [woListSchemaKeys.appointmentId]: object.appointmentId,
            [woListSchemaKeys.workOrderNumber]: object.workOrderNumber,
            [woListSchemaKeys.ifsWorkOrderStatusId]:
              object.ifsWorkOrderStatusId,
            [woListSchemaKeys.workOrderColor]: object.workOrderColor,
            [woListSchemaKeys.licensePlateNumber]: object.licensePlateNumber,
            [woListSchemaKeys.model]: object.model,
            [woListSchemaKeys.brandName]: object.brandName,
            [woListSchemaKeys.logo]: object.logo,
            [woListSchemaKeys.customerName]: object.customerName,
            [woListSchemaKeys.mobileNo]: object.mobileNo,
            [woListSchemaKeys.description]: object.description,
            [woListSchemaKeys.plannedTime]: object.plannedTime,
            [woListSchemaKeys.bookedDate]: object.bookedDate,
            [woListSchemaKeys.updatedDate]: object.updatedDate,
            [woListSchemaKeys.workOrderStatus]: object.workOrderStatus,
            [woListSchemaKeys.dealer]: object.dealer,
            [woListSchemaKeys.appointmentGuid]: object.appointmentGuid,
          };
          printLog("check object````", inputData);
          Realm.create(schemas.wo_list_table, inputData);
        });
        dataWritten = true;
      });
    } catch (error) {
      dataWritten = false;
      errorMsg = error;
    }
    if (dataWritten) {
      alert("check data added true");
      // return Promise.resolve(true);
    } else {
      alert(" opppppp  check data not added ");
      // return Promise.reject(
      //   errorMsg != undefined ? errorMsg : "Problem writing data"
      // );
    }

**Comment by kneth** [comment]
In order to use Reanimated v2.9.1, you will need to upgrade to Realm JavaScript v11.0.0 or later.

**Comment by takameyer** [comment]
@Vivaan7 You are not the first to report this problem.  See our [closed issues relating to `react-native-reanimated`](https://github.com/realm/realm-js/issues?q=is%3Aissue+is%3Aclosed+reanimated+).  You can see that many others also did the upgrade to fix the problem (see #4399).  I know it's a painful upgrade, but it is really the only way forward.  There was a lot of hard work put in to fix this issue in Realm, but it requires an updated React Native.

**Comment by Vivaan7** [comment]
@takameyer @kneth  thanks for you support now i am not using this library anymore !
